src/widget/block.rs

```diff
@@ -209,3 +209,38 @@ impl Widget for Block {
         self.content.lifecycle(cx, event);
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use ratatui::layout::Size;
+    use ratatui::style::Color;
+
+    use crate::test_helper::render_widget;
+    use crate::widget::Text;
+
+    use super::*;
+
+    #[test]
+    fn simple_text_test() {
+        let content = Text {
+            text: "some text".into(),
+            style: Style::default().fg(Color::Red),
+        };
+        let mut sut = Block::new(
```

Comment:
Just out of interest what does `sut` mean (as it's not immediately clear)?

Reply:
"system under test" I thought it was common enugh to just use it.
This way the central subject stands out a bit in a test.